cli/flox/src/commands/containerize/mod.rs

```diff
@@ -85,12 +90,15 @@ impl Containerize {
         let lockfile: Lockfile = env.lockfile(&flox)?.into();
         let manifest = lockfile.manifest;
         let mode = manifest.options.activate.mode.unwrap_or_default();
-
         let source = if std::env::consts::OS == "linux" {
             let container_config = manifest
                 .containerize
                 .and_then(|c| c.config)
-                .map(|c| c.into());
+                .or_else(|| should_extend_config(&self.label).then(Default::default))
+                .map(|mut c| {
+                    extend_config(&self.label, &mut c);
```

Comment:
nit: I think in some cases this adds a `Some([])` for labels when we could just leave it as None